src/analysis/retail/evoker/devastation/modules/guide/CoreRotation.tsx

```diff
@@ -25,7 +25,7 @@ export function CoreRotation({ modules, info }: GuideProps<typeof CombatLogParse
       </p>
       <AplSectionData checker={AplCheck.check} apl={AplCheck.apl()} />
       <hr />
-      <p>
+      <div>
```

Comment:
also note: `<p>` and `<div>` do not have the same styles. I checked several specs to confirm it didn't make them look (super) bad but this is generally not a 1-1 swap (and the browser *may* allow invalid nesting in some cases)

Reply:
Yes. But understood it needed to be a block element. Inspecting the guides looks like there was no default `<p>` `margin-top`/`margin-bottom` to account for.

It's unfortunate that all guides are written with `<p>` tags as default when people who are not seasoned frontend/react developers know about the limitiations/rules surrounding `<p>` tags. Guides could probable exposed a React component to wrap all blocks in with some padding and whatever, and not use any default tags tbh.

Reply:
You can blame me for a lot of the use of `<p>`. I used it a lot in some of the early stuff that got used as a reference